case-study/sqls/working-hour-distribution/sql

```diff
@@ -0,0 +1 @@
+SELECT COUNT(*) AS count, toDayOfWeek(created_at) AS dayOfWeek, toHour(created_at) AS hour FROM {{table}} WHERE repo_id IN [{{repoIdList}}] GROUP BY hour, dayOfWeek
```

Comment:
With this SQL you may get the working hour distribution for all repos in the case study, but I think we should have distribution for each repo.